flink-core/src/main/java/org/apache/flink/configuration/description/TextElement.java

```diff
@@ -53,6 +55,16 @@ public static TextElement text(String text) {
 		return new TextElement(text, Collections.emptyList());
 	}
 
+	/**
+	 * Tries to format the text as code.
+	 *
+	 * @return text element with applied formatting
+	 */
+	public TextElement formatAsCode() {
```

Comment:
alternatively we could add an explicit `Code` `InlineElement`.

Reply:
I thought about it. The reason why I chose it as part of `TextElement` is that this way we could implement styles as e.g. div classes in a single node. With explicit element we would always need to model it as node hierarchy.

Reply:
At the very least I would add a static `code(String)` method to make it more readable. i.e.
```
 	public static TextElement code(String text) {
		return new TextElement(text, TextStyle.CODE, Collections.emptyList());
	}
```